app/models/Product.scala

```diff
@@ -1,3 +1,21 @@
 package models
 
+import play.api.libs.json.{JsMacroImpl, Json, OFormat}
+
+import scala.language.experimental.macros
+
+trait ImplicitJsonFormat
+trait JsonImplicits {
+  implicit def implicitJsonFormat[A <: ImplicitJsonFormat]: OFormat[A] = macro JsMacroImpl.implicitConfigFormatImpl[A]
+}
+
 case class Product(service: String, environment: String, serverTime: String)
+
+/** Object Companion for serialization
+ *
+ */
```

Comment:
This is obvious, you can remove this comment

Reply:
You are right, will remove it!